docs/configuration.md

```diff
@@ -2248,8 +2248,8 @@ should be included on Spark's classpath:
 * `hdfs-site.xml`, which provides default behaviors for the HDFS client.
 * `core-site.xml`, which sets the default filesystem name.
 
-The location of these configuration files varies across CDH and HDP versions, but
-a common location is inside of `/etc/hadoop/conf`. Some tools, such as Cloudera Manager, create
+The location of these configuration files varies across Hadoop versions, but
```

Comment:
Hm, I guess one issue I've realized now is that it doesn't really vary across versions of Hadoop but could vary according to packaging and distribution. If you change it again, also consider fixing the existing "mechanisms" typo in line 2253